src/gsi/gsd_update_mod.f90

```diff
@@ -530,14 +533,14 @@ subroutine gsd_update_th2(tinc,it)
   real(r_kind) :: dth2, work_prsl,work_prslk
 
   real(r_kind),dimension(:,:  ),pointer:: ges_ps =>NULL()
-  real(r_kind),dimension(:,:  ),pointer:: ges_th2=>NULL()
+  real(r_kind),dimension(:,:  ),pointer:: ges_t2m=>NULL()
   real(r_kind),dimension(:,:,:),pointer:: ges_q  =>NULL()
 
 !*******************************************************************************
 !
 ! 2-m temperature
 !  do it=1,nfldsig
-     call gsi_bundlegetpointer(gsi_metguess_bundle(it),'th2m',ges_th2,ier)
+     call gsi_bundlegetpointer(gsi_metguess_bundle(it),'t2m',ges_t2m,ier)
```

Comment:
This change will also work for global applications if gsd_update_t2m() is called , right?